### PR TITLE
Issue #2450: Restore admin bar links to manage display pages.

### LINKED ISSUE #2450: [UX] Add back admin bar links to manage display sub-pages
While working on https://github.com/backdrop/backdrop-issues/issues/315 we removed the menu router items for the sub-tabs on manage displays. This also effectively removed them from the admin bar. We should add them back or there will be a little UX regression.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2450

### CODE REVIEW

File: core/modules/field_ui/field_ui.admin.inc [hunk 1]
```diff
@@ -2333,7 +2333,11 @@ function field_ui_view_mode_delete_form_submit($form, &$form_state) {
  * Access callback: Checks the users access and token to enable a display mode.
  */
 function field_ui_view_mode_enable_access() {
-  return user_access('administer view modes') && backdrop_valid_token($_GET['token'], 'view_mode_enable');
+  $token_check = TRUE;
```
Comment (opi) [inline core/modules/field_ui/field_ui.admin.inc:2479]: default should be FALSE, isn't it ?

**Comment by jenlampton** [comment]
closing for new sandbox.

**Comment by jenlampton** [comment]
new sandbox

File: core/modules/field_ui/field_ui.admin.inc (hunk 1 above)
Comment (jenlampton) [reply core/modules/field_ui/field_ui.admin.inc:2479]: I don't know. Do we deny access if `$_GET['token']` isn't set? I was guessing we would only deny access if a token WAS set but it wasn't valid.

**Comment by stpaultim** [comment]
@jenlampton - Sandbox not working.

File: core/modules/admin_bar/admin_bar.inc [hunk 2]
```diff
@@ -490,11 +490,9 @@ function admin_bar_links_menu($tree) {
     // Remove description to prevent mouseover tooltip clashes.
     unset($data['link']['localized_options']['attributes']['title']);
 
-    // Make action links (typically "Add ...") appear first in dropdowns.
-    // They might appear first already, but only as long as there is no link
-    // that comes alphabetically first (e.g., a node type with label "Ad").
+    // Make action links (typically "Add ...") appear last in dropdowns.
```
Comment (klonos) [inline core/modules/admin_bar/admin_bar.inc:493]: @jenlampton I believe that we are addressing this one separately in https://github.com/backdrop/backdrop-issues/issues/4586, so this change here needs to be reverted please.

File: core/modules/file/file.module [hunk 3]
```diff
@@ -205,14 +205,10 @@ function file_menu() {
...
   );
-  // File Configuration
   $items['admin/structure/file-types/list'] = array(
-    'title' => 'File types',
-    'description' => 'Manage settings for the type of files used on your site.',
-    'page callback' => 'file_list_types_page',
-    'access arguments' => array('administer file types'),
+    'title' => 'List file types',
     'type' => MENU_DEFAULT_LOCAL_TASK,
-    'file' => 'file.admin.inc',
```
Comment (klonos) [inline core/modules/file/file.module:215]: These changes seem irrelevant - they need to be reverted I believe.

**Comment by jenlampton** [comment]
Closing in favor of @argiepiano 's newer pr
